payments/mobile-pay/capture.md

```diff
@@ -1,17 +0,0 @@
-title: MobilePay Payments – Capture
```

Comment:
Why is `payments/mobile-pay/capture.md` is deleted?

Reply:
Oh, not sure how this happend. I will put it back in

Reply:
oh right, because the right file is /payment-instrument/mobile-pay/capture.md 
I dont know why it was still a payments file. This should be deleted